src/auth/registration/AddressField.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, Typography } from "@mui/material";
+import { FormikTextField } from "form/FormikHelpers";
+import React from "react";
+import { CountryPicker } from "components/Country/CountryPicker";
+
+export function AddressField() {
+  return (
+    <React.Fragment>
+      <Typography variant="body1" gutterBottom>
+        We are excited to have the opportunity to work with you! The last bit we
+        need is some address info for your organization. This will allow us to
+        set up the necessary financial arrangements and ensure a seamless
+        partnership.
+      </Typography>
+
+      <FormikTextField name="address.street1" label="Street address" />
```

Comment:
If possible, could we make sure we set the [appropriate html attributes on these fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete) so that e.g. autofill of an address from 1password works as expected?